### ISSUE TITLE
Chessboard mounting

### ISSUE THREAD

**Issue by AwsKhalil** [mfxox/ILCC#26]
Thank you for your great work.

for chessboard detection from point cloud, does the chessboard need to be mounted on a stand or can a person hold it ? would that affect the detection of the chessboard ?

**Comment by ValterZoldhegyi** [comment]
It is beneficial to use a stand. The reasons for that:
1. The reflected points from the person may cause wrong segmentation results: the points from the person and the chessboard may be clustered to one segment. 
2. If the chessboard is not standstill in the hand of the person, the camera and the lidar should record the chessboard in the same time (synchronised).

**Comment by AwsKhalil** [comment]
Thank you. I figured that your first point is my problem.  I couldn't perform the measurement again so here is how I solved it ( for anyone facing similar problem in the future) :
1. run the 3 scripts normally (img_corner_est, pcd_corner_est, LM_opt) 
2. based on the results you get after running LM_opt script, find the odd images which show that there is a corner between the person and the chessboard and filter them out along with their point cloud from the original image,pointcloud pairs you had in the folders img and pcd 
3. rename the images and the point clouds and run the 3 scripts again. 

Note: Make sure that the intrinsic parameters ('instrinsic_para'  in the config file) are accurate.
I attached an example of an odd image
![Annotation 2020-08-27 182038](https://user-images.githubusercontent.com/53193289/91468277-0c73b400-e892-11ea-9456-bfa6942a4fac.png)
